### PR TITLE
Show detail Playone

### PR DESCRIPTION
**Pull request by pokk**
as the title.

### CODE REVIEW

**Comment by pokk** [comment]
### TODO

Add the providing objects in the DI module.

File: mobile-ui/src/main/res/layout/fragment_playone_detail.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="1.5dp"
+        android:layout_marginTop="16dp"
+        android:background="@color/white"
+        app:layout_constraintBottom_toTopOf="@id/rv_participation"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView2"/>
+
+    <android.support.v7.widget.RecyclerView
```
Comment (lekaha) [inline mobile-ui/src/main/res/layout/fragment_playone_detail.xml:85]: I forgot what this list used for??

File: mobile-ui/src/main/res/layout/fragment_playone_detail.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    android:layout_height="match_parent">
+
+    <include
+        android:id="@+id/cl_playone"
+        layout="@layout/item_playone_constraint"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+
+    <com.google.android.gms.maps.MapView
```
Comment (lekaha) [inline mobile-ui/src/main/res/layout/fragment_playone_detail.xml]: I remember we were using static map (image), right?

File: mobile-ui/src/main/res/layout/fragment_playone_detail.xml (hunk 1 above)
Comment (pokk) [reply mobile-ui/src/main/res/layout/fragment_playone_detail.xml:85]: I added it and for the team's quantity.

File: mobile-ui/src/main/res/layout/fragment_playone_detail.xml (hunk 2 above)
Comment (pokk) [reply mobile-ui/src/main/res/layout/fragment_playone_detail.xml]: Yes! I forgot it. Thank you for remaindering it.

File: mobile-ui/src/main/java/com/playone/mobile/ui/playone/PlayoneDetailFragment.kt [hunk 3]
```diff
@@ -86,10 +88,9 @@ class PlayoneDetailFragment : BaseInjectingFragment() {
     }
 
     private fun setupRecycler() {
-
-//        rv_playone_list.apply {
-//            layoutManager = LinearLayoutManager(activity, LinearLayoutManager.HORIZONTAL, false)
-//            adapter = playoneAdapter
-//        }
+        rv_participation.apply {
```
Comment (lekaha) [inline mobile-ui/src/main/java/com/playone/mobile/ui/playone/PlayoneDetailFragment.kt]: `rv_participation` is which ID you use for the view not `rv_playone_list.apply`

**Pull request merged**